devservices/config.yml

```diff
@@ -0,0 +1,40 @@
+x-sentry-service-config:
+  version: 0.1
+  service_name: uptime-checker
+  dependencies:
+    kafka:
+      description: Shared instance of kafka used by sentry services
+      remote:
+        repo_name: sentry-shared-kafka
+        branch: main
+        repo_link: git@github.com:getsentry/sentry-shared-kafka.git
+    redis:
+      description: Shared instance of redis used by sentry services
+      remote:
+        repo_name: sentry-shared-redis
+        branch: main
+        repo_link: git@github.com:getsentry/sentry-shared-redis.git
+  modes:
+    default: [redis]
+    sentry: [kafka, redis]
+services:
+  uptime-checker:
+    image: ghcr.io/getsentry/uptime-checker:nightly
+    command:
+      - run
+    environment:
+      UPTIME_CHECKER_RESULTS_KAFKA_CLUSTER: kafka-kafka-1:9092
+      UPTIME_CHECKER_CONFIGS_KAFKA_CLUSTER: kafka-kafka-1:9092
```

Comment:
This is actually pretty much unused now. I think we can leave this one out

Reply:
actually let's just leave it. We can always clean up later

Reply:
It's needed for https://github.com/getsentry/sentry/pull/102878 to get uptime-chcker to use sentry's shared kafka